extensions/test/NETCore.SetupTests/ConfigurationTests.cs

```diff
@@ -152,7 +152,7 @@ public void GetClientConfigSettingsCaseInsensitiveTest()
             Assert.Equal("us-east-1", client.Config.AuthenticationRegion);
             Assert.Equal(DefaultConfigurationMode.Standard, client.Config.DefaultConfigurationMode);
             Assert.Equal(RequestRetryMode.Standard, client.Config.RetryMode);
-            Assert.Equal("https://localhost:9021", client.Config.ServiceURL);
+            Assert.Equal("https://localhost:9021/", client.Config.ServiceURL);
```

Comment:
This test started failing once I updated the Core version due to a change last year to make sure the ServiceUrl always has a trailing slash. https://github.com/aws/aws-sdk-net/commit/9d23793f1b8e3d36d48c40ba8a393fccea90c05d